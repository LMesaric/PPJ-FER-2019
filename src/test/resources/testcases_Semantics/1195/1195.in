<prijevodna_jedinica>
 <vanjska_deklaracija>
  <definicija_funkcije>
   <ime_tipa>
    <specifikator_tipa>
     KR_INT 17 int
   IDN 17 main
   L_ZAGRADA 17 (
   KR_VOID 17 void
   D_ZAGRADA 17 )
   <slozena_naredba>
    L_VIT_ZAGRADA 17 {
    <lista_deklaracija>
     <lista_deklaracija>
      <deklaracija>
       <ime_tipa>
        <specifikator_tipa>
         KR_INT 19 int
       <lista_init_deklaratora>
        <init_deklarator>
         <izravni_deklarator>
          IDN 19 a
          L_UGL_ZAGRADA 19 [
          BROJ 19 2024
          D_UGL_ZAGRADA 19 ]
         OP_PRIDRUZI 19 =
         <inicijalizator>
          <izraz_pridruzivanja>
           <log_ili_izraz>
            <log_i_izraz>
             <bin_ili_izraz>
              <bin_xili_izraz>
               <bin_i_izraz>
                <jednakosni_izraz>
                 <odnosni_izraz>
                  <aditivni_izraz>
                   <multiplikativni_izraz>
                    <cast_izraz>
                     <unarni_izraz>
                      <postfiks_izraz>
                       <primarni_izraz>
                        NIZ_ZNAKOVA 19 "dfvgfdgdfsgsdfgsgsdfgdsgsgdsfgdgdfgfdgfdgfgfdsgdfsgsdgfsdghgdfhgffgsfgsgfsfdgdgdfgfdgfdgdfgdfgfgdgsdgsdgdssfgsddfvgfdgdfsgsdfgsgsdfgdsgsgdsfgdgdfgfdgfdgfgfdsgdfsgsdgfsdghgdfhgffgsfgsgfsfdgdgdfgfdgfdgdfgdfgfgdgsdgsdgdssfgsddfvgfdgdfsgsdfgsgsdfgdsgsgdfgdgdfgfdgfdgfgfdsgdfsgsdgfsdghgdfhgffgsfgsgfsfdgdgdfgfdgfdgdfgdfgfgdgsdgsdgdssfgsddfvgfdgdfsgsdfgsgsdfgdsgsgdsfgdgdfgfdgfdgfgfdsgdfsgsdgfsdghgdfhgffgsfgsgfsfdgdgdfgfdgfdgdfgdfgfgdgsdgsdgdssfgsddfvgfdgdfsgsdfgsgsdfgdsgsgdsfgdgdfgfdgfdgfgfdsgdfsgsdgfsdghgdfhgffgsfgsgfsfdgdgdfgfdgfdgdfgdfgfgdgsdgsdgdssfgsddfvgfdgdfsgsdfgsgsdfgdsgsgdsfgdgdfgfdgfdgfgfdsgdfsgsdgfsdghgdfhgffgsfgsgfsfdgdgdfgfdgfdgdfgdfgfgdgsdgsdgdssfgsdvfbgvfdgdvfdfvgfdgdfsgsdfgsgsdfgdsgsgdsfgdgdfgfdgfdgfgfdsgdfsgsdgfsdghgdfhgffgsfgsgfsfdgdgdfgfdgfdgdfgdfgfgdgsdgsdgdssfgsddfvgfdgdfsgsdfgccsdddvdfsavafdvadfvsgsdfgdsgsgdsfgdgdfgfdgfdgfgfdsgdfsgsdgfsdghgdfhgffgsfgsgfsfdgdgdfgfdgfdgdfgdfgfgdgsdgsdtgfgfggbvgfbthsfgbdhgshgdssfgsddfvgfdgdfsgsdfgsgsdfgdsgsgdsfgdgfdgfdgfgfdsgdfsgsdgfsdghgdfhgfggsdgagadfagghgadgdf"
       TOCKAZAREZ 19 ;
     <deklaracija>
      <ime_tipa>
       <specifikator_tipa>
        KR_CHAR 20 char
      <lista_init_deklaratora>
       <init_deklarator>
        <izravni_deklarator>
         IDN 20 b
        OP_PRIDRUZI 20 =
        <inicijalizator>
         <izraz_pridruzivanja>
          <log_ili_izraz>
           <log_i_izraz>
            <bin_ili_izraz>
             <bin_xili_izraz>
              <bin_i_izraz>
               <jednakosni_izraz>
                <odnosni_izraz>
                 <aditivni_izraz>
                  <multiplikativni_izraz>
                   <cast_izraz>
                    <unarni_izraz>
                     <postfiks_izraz>
                      <primarni_izraz>
                       ZNAK 20 'b'
      TOCKAZAREZ 20 ;
    <lista_naredbi>
     <naredba>
      <naredba_skoka>
       KR_RETURN 21 return
       <izraz>
        <izraz_pridruzivanja>
         <log_ili_izraz>
          <log_i_izraz>
           <bin_ili_izraz>
            <bin_xili_izraz>
             <bin_i_izraz>
              <jednakosni_izraz>
               <odnosni_izraz>
                <aditivni_izraz>
                 <multiplikativni_izraz>
                  <cast_izraz>
                   <unarni_izraz>
                    <postfiks_izraz>
                     <primarni_izraz>
                      BROJ 21 0
       TOCKAZAREZ 21 ;
    D_VIT_ZAGRADA 22 }
