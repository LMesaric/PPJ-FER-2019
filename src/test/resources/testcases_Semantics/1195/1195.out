<izravni_deklarator> ::= IDN(19,a) L_UGL_ZAGRADA(19,[) BROJ(19,2024) D_UGL_ZAGRADA(19,])
